### DISCUSSION TITLE
Ideal ratio of temporary storage to final storage?

### DISCUSSION THREAD

**Discussion by Zerophase** [Chia-Network/chia-blockchain#5242] [Q&A] [answered]
So according to my understanding Chia requires 370GB of space for each plot, and 100GB for the final storage plot. So, if I have 20TB of SSDs for temp I need at least 6TB of HDD for storage? Is my math right? This could really bring down the cost of a rig.

**Comment by knightblackm** [comment, marked as answer]
no, you need about 240Gb for temp in last of phase 2 and its reduce then, after ploting completed, temp removed and you have 240 space on temp ssd (for k32). you need ssd for build temp file only!

**Reply by Zerophase** [reply]
So, an 8tb SSD should be about enough to hold the final plots, coming from a 20tb drive doing plotting?

**Reply by knightblackm** [reply]
don't use ssd for holding, if your cpu and ram is high, u can use parallel option and make plot faster. build a rig contain 16Tb hdd high speed (for copying plot file to hdd) and u plot hard faster!

**Reply by knightblackm** [reply]
you can parallel 9-8 plot with 2Tb nvme ssd.

**Reply by Zerophase** [reply]
Yeah, I get that. What I'm asking is is it better to dump the final plots on fast storage, and then use other scripts to move it to slow storage. I'm trying to keep all my disks writing, while minimizing the downtime from transferring to the final spot. Writing to an HDD seems like a bottleneck another drive could handle better than the plotting drives.
